src/main/java/io/reactivex/Observable.java

```diff
@@ -13216,21 +13201,17 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
     }
 
     /**
-     * Returns an Observable that only emits those items emitted by the source ObservableSource that are not followed
-     * by another emitted item within a specified time window, where the time window is governed by a specified
-     * Scheduler.
+     * Alias to {@link #debounce(long, TimeUnit, Scheduler)}
```

Comment:
Please keep the original sentence first.

Reply:
done